share/functions/__fish_git_prompt.fish

```diff
@@ -478,7 +478,7 @@ function __fish_git_prompt_informative_status
     set -l dirtystate (math (count $changedFiles) - (count (echo $changedFiles | grep "U")))
     set -l invalidstate (count (echo $stagedFiles | grep "U"))
     set -l stagedstate (math (count $stagedFiles) - $invalidstate)
-    set -l untrackedfiles (count (command git ls-files --others --exclude-standard))
+    set -l untrackedfiles (command git ls-files --others --exclude-standard | wc -l | string trim)
```

Comment:
Why is `| string trim` required? The `wc -l` portion of that command will return a single line that the fish sub-command will turn into a single value since it strips (i.e., ignores) a trailing newline.